docs/theming/high-contrast-mode.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Hide Contrast Mode
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<head>
+  <title>High Contrast Mode to Increase Color Contrast</title>
+  <meta
+    name="description"
+    content="Developers are adding high contrast mode CSS on native applications to support their user preferences. Read to learn more about high contrast color schemes for Ionic apps."
+  />
+</head>
+
+Ionic offers themes with increased contrast for users with low vision. These themes work by amplifying the contrast between foreground content, such as text, and background content, such as UI components. Ionic provides both light and dark variants for achieving high contrast.
+
+## Overview
+
+The default theme in Ionic provides [Ionic colors](./colors) that meet [Level AA color contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) as defined by Web Content Accessibility Guidelines (WCAG) when used with the appropriate contrast color. The [Ionic colors](./colors) in the high contrast theme have been updated to meet [Level AAA color contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced.html) when used with the appropriate contrast color. Notably, improvements have been made to the contrast of UI components, including border, text, and background colors. However, it's important to note that within the high contrast theme, priority is given to text legibility. This means that if adjusting the contrast of a UI component against the page background would significantly compromise the contrast between the component's text and its background, the contrast of the UI component background will remain unchanged.
+
+## Enabling High Contrast Theme
+
+There are three provided ways to enable the high contrast theme in an app: **always**, based on **system** settings, or by using a **class**.
+
+### Always
+
+The high contrast theme can be enabled by importing the following stylesheet in the appropriate files. This approach will enable the high contrast theme regardless of the system settings for contrast preference.
+
+<Tabs groupId="framework" defaultValue="angular" values={[{ value: 'angular', label: 'Angular' }, { value: 'javascript', label: 'Javascript' }, { value: 'react', label: 'React' }, { value: 'vue', label: 'Vue' }]}>
+
+<TabItem value="angular">
+
+```typescript
```

Comment:
```suggestion
```css
```

Reply:
This is technically a JS import, since devs would to it in `main.ts` or something. Should this be `javascript` instead for the code block?

Reply:
In Angular it is in the CSS though: https://github.com/ionic-team/starters/blob/feature-8.0/angular/base/src/global.scss

Reply:
Oh 🤦 I have my frameworks backwards. In other frameworks it's JS, but not in Angular. I'll fix

Reply:
https://github.com/ionic-team/ionic-docs/pull/3431/commits/7e623d6f02c61d40e92bfa588e2a9647c60a11f2